### PR TITLE
Next song backend fixes #128

### LINKED ISSUE #128: Fix the play history.

### PR DESCRIPTION
**Pull request by JeremybellEU**
In relation to https://github.com/MoodCat/MoodCat.me-Frontend/pull/44

### CODE REVIEW

**Comment by Azkle** [comment]
Can be merged.

File: src/main/java/me/moodcat/backend/RoomBackend.java [hunk 1]
```diff
@@ -226,7 +231,6 @@ public void sendMessage(final ChatMessage chatMessage) {
          * Merge the changes of the instance in the database.
          */
         protected void merge() {
-            log.info("Merging changes in room {}", this);
```
Comment (JWGmeligMeyling) [inline src/main/java/me/moodcat/backend/RoomBackend.java]: Please keep the log for a while. Maybe decrease the log level to debug so we can filter it in production.

File: src/main/java/me/moodcat/database/entities/Room.java [hunk 2]
```diff
@@ -62,12 +60,7 @@
     /**
      * Songs to be played.
      */
-    @ManyToMany(fetch = EAGER)
-    @JoinTable(name = "room_play_queue", joinColumns = {
-            @JoinColumn(name = "room_id", referencedColumnName = "id")
-    }, inverseJoinColumns = {
-            @JoinColumn(name = "song_id", referencedColumnName = "id")
-    })
+    @Transient
```
Comment (JWGmeligMeyling) [inline src/main/java/me/moodcat/database/entities/Room.java]: This should not be the way we want to do this. You've just dropped the columns? So then just remove the attribute. A transient column makes no sense. If we want the history and playqueue in the database, there should be tables and they should be eagerly fetched manytomany relations. If we dont want this in the database, then we should remove the attributes from the entity alltogher. This is no fix but rather a clean up the entire model :P

**Comment by JWGmeligMeyling** [comment]
It's unclear to me what you're trying to achieve here. It seems you made things to work by just removing all the things that the Room entity was supposed to do? :P

**Pull request closed without merging**